_pages/release_notes.md

```diff
@@ -8,7 +9,121 @@ image:
     teaser: /icons/algo.svg
 ---
 
-# Q1 2020 Enterprise Release Notes #
+# Q2 2020 Release Notes #
+
+
+### Introduction ###
+
+We are pleased to present several new features and upgrades this quarter, most notably around security and governance of machine learning systems. This release showcases advanced security options that enable customers to operate Algorithmia in restrictive environments, including AWS C2S, AWS GovCloud, VMware, authenticated proxies, customer-provided OS images, private Docker hubs, private dependency mirrors, and private certificate authorities. 
```

Comment:
i think we should say `private Docker registries` not `private Docker hubs`

Reply:
Updated.